src/blockchain/log-processors/designated-report-submitted.ts

```diff
@@ -3,7 +3,35 @@ import * as Knex from "knex";
 import { FormattedLog, ErrorCallback } from "../../types";
 
 export function processDesignatedReportSubmittedLog(db: Knex, augur: Augur, trx: Knex.Transaction, log: FormattedLog, callback: ErrorCallback): void {
-    console.log("TODO: DesignatedReportSubmitted");
-    console.log(log);
-    callback(null);
+    // @achapman: "The designated reporter purchases Stake Tokens in order to do the Designated Report"
+    // Should designatedReportSubmitted include StakedToken address?
+
+    // for phase, sqlite lacks an ENUM, but the idea here is that:
+    //  phase:null = no report submitted
+    //  phase:0 = designated report submitted
+    //  phase:1 = round1
+    //  phase:2 = round2
+    // if that seems reasonable, i'll update PR to import the enum and use name below
+    const marketStateDataToInsert: { [index: string]: string | number | boolean } = {
+        marketID: log.market,
+        phase: 0,
+        isDisputed: false,
+        blockNumber: log.blockNumber
+    };
+
+    db.transacting(trx).insert(marketStateDataToInsert).returning("marketStateID").into("market_state").asCallback((err: Error | null, marketStateID: Array<number>): void => {
+        if (err) return callback(err);
+        if (!marketStateID) return callback(new Error("Failed to generate new marketStateID for marketID:" + log.market));
```

Comment:
Is it possible for marketStateID to be an empty array here?  If so, should change this conditional to `if (!marketStateID || !marketStateID.length)`.

Reply:
good call, my Python is showing...  in Python, `[]` is false.